# origin: akpm@osdl.org (BitKeeper)
# cset: 4237a9a6ISBXvG9UQAZGLns5WgKYGA
# URL: http://linux.bkbits.net:8080/linux-2.6/cset@4237a9a6ISBXvG9UQAZGLns5WgKYGA
# inclusion: upstream (backport)
# description: [PATCH] dvb pll warning fix
# revision date: 2005/03/15 19:36:06-08:00
#
diff -Naru a/drivers/media/dvb/frontends/dvb-pll.h b/drivers/media/dvb/frontends/dvb-pll.h
--- a/drivers/media/dvb/frontends/dvb-pll.h	2005-04-01 19:56:08 -08:00
+++ b/drivers/media/dvb/frontends/dvb-pll.h	2005-04-01 19:56:08 -08:00
@@ -14,7 +14,7 @@
 		u32 stepsize;
 		u8  cb1;
 		u8  cb2;
-	} entries[0];
+	} entries[9];
 };
 
 extern struct dvb_pll_desc dvb_pll_thomson_dtt7579;
# This is a BitKeeper generated diff -Nru style patch.
#
# ChangeSet
#   2005/03/15 19:36:06-08:00 akpm@osdl.org 
#   [PATCH] dvb pll warning fix
#   
#     CC [M]  drivers/media/dvb/frontends/dvb-pll.o
#   drivers/media/dvb/frontends/dvb-pll.c:38: warning: excess elements in array initializer
#   drivers/media/dvb/frontends/dvb-pll.c:38: warning: (near initialization for `dvb_pll_thomson_dtt7579.entries')
#   
#   
#   Signed-off-by: Andrew Morton <akpm@osdl.org>
#   Signed-off-by: Linus Torvalds <torvalds@osdl.org>
# 
# drivers/media/dvb/frontends/dvb-pll.h
#   2005/03/15 06:30:27-08:00 akpm@osdl.org +1 -1
#   dvb pll warning fix
# 
